applications/fishing-map/src/data/config.ts

```diff
@@ -46,3 +49,45 @@ export const sources: SelectOption[] = [
 ]
 
 export const DATE_FORMAT = 'yyyy/MM/dd'
+
+export const timebarEventsOptions: SelectOption[] = [
```

Comment:
```suggestion
export const TIMEBAR_EVENT_OPTIONS: SelectOption[] = [
```
Please use SCREAMING_SNAKE_CASE by convention when setting readonly consts